src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,93 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_SIZE = 16;
+    private static final int RESIZE_CONSTANT = 2;
+    private static final float LOAD_FACTOR = 0.75f;
+    private int size;
+    private Node<K, V>[] arrayNode;
+
+    public MyHashMap() {
+        arrayNode = new Node[DEFAULT_SIZE];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        resize();
+        putElement(key, value);
     }
 
     @Override
     public V getValue(K key) {
+        int index = (key == null) ? 0 : getIndex(key);
```

Comment:
`getIndex `must return a valid value 
```suggestion
        int index = getIndex(key);
```